pyani/scripts/average_nucleotide_identity.py

```diff
@@ -871,7 +871,8 @@ def process_arguments(args: Optional[Namespace]) -> Namespace:
     shows the version and exits.
     """
     # Catch execution with no arguments
-    if len(sys.argv) == 1:
+    # But only for the `pyani` executable (`pytest` was also being caught here)
```

Comment:
I don't understand why `endswith()` is used in preference to `==`. What is your thinking, here?

Also, the `pyani` command would not execute`average_nucleotide_identity.py` - the command for this is `average_nucleotide_identity.py` (see [this line](https://github.com/widdowquinn/pyani/blob/62949c49d971aa33e413821905565368563ad056/setup.py#L96)), so should this not be testing for `average_nucleotide_identity.py` rather than `pyani`?

Reply:
For the `==` part, see my response below.

> Also, the `pyani` command would not execute `average_nucleotide_identity.py`

Yes, you're right; that was a mistake on my part.

Reply:
So long as you make the change such that v0.2 tests for `average_nucleotide_identity.py` this part of the conversation can be resolved.